### PR TITLE
{data}[iimpi/2019a] HDF5 v1.10.5

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/h/HDF5/HDF5-1.10.5-intel-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name = 'HDF5'
+version = '1.10.5'
+
+homepage = 'https://portal.hdfgroup.org/display/support'
+description = """HDF5 is a data model, library, and file format for storing and managing data.
+ It supports an unlimited variety of datatypes, and is designed for flexible
+ and efficient I/O and for high volume and complex data."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
```
Comment (boegel) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.10.5-intel-2019a.eb]: @akesandgren HDF5 doesn't use BLAS/LAPACK, so we can pull this down to the `iimpi` toolchain?

Comment (akesandgren) [reply easybuild/easyconfigs/h/HDF5/HDF5-1.10.5-intel-2019a.eb]: Yeah should be doable.... will try

Comment (akesandgren) [reply easybuild/easyconfigs/h/HDF5/HDF5-1.10.5-intel-2019a.eb]: Done

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 1 out of 7 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/05feefad738f55f2710e55047e990d8c for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/14de2c5778c24f2b3965fda75d82429d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d027a2be9e7804f044d342b730046fca for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2077.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/92f8cafb56ab5f06bf1a7af0d891f184 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node3134.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/abea23e989bf1c8036114e4399dee493 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**